README.md

```diff
@@ -0,0 +1,80 @@
+# Chinese Newcomers Service Center
+This project is designed and developed by a team of UC Berkeley students through one of [Cal Blueprint](https://calblueprint.org/)'s project teams during the 2022-23 academic year. 
+
+Learn more about [Chinese Newcomers Service Center](https://chinesenewcomers.org/) and [Cal Blueprint](https://calblueprint.org/).
+
+---
+## Getting Started
+
+### Prerequisites
+
+Check your installation of `npm` and `node`:
+
+```sh
+node -v
+npm -v
+```
+
+This project requires node version 16.17.0.
```

Comment:
@surajr10 this is based on the fact that i couldn't get yarn install to work with 12, 14, or 18, and @anniehuo's worked on 16.17.0 If you know other versions also work, feel free to suggest and I'll update.